docs/modules/ROOT/pages/getting-started.adoc

```diff
@@ -1132,6 +1132,8 @@ You can login in with the Crowd admin user you set up earlier.
 
 === Add shared images
 
+NOTE: This section is not required anymore from OpenDevStack 2.0
```

Comment:
Antora is versioned, please kill the whole section.

Reply:
There was another place where uploading templates is mentioned where it says it only applies before 1.2.x or similar. I could dig out the link for that.  So perhaps the docs needs some review to remove version references?

Reply:
Would be nice to clean that up, yes.